contracts/src/starknet/rewaave/token_bridge.cairo

```diff
@@ -47,7 +47,7 @@ func minted_rewards(l2_reward_token : felt, account : felt, amount : Uint256):
 end
 
 @event
-func bridged_rewards(l2_token : felt, acocunt : felt, amount : Uint256):
+func bridged_rewards(l1_recipient : felt, amount : Uint256):
```

Comment:
This should also have the l2 user who bridged the rewards

Reply:
Since there is no strict relationship between the l1 and l2 user they should both be indicated. @swapnilraj and I discussed this offline and we'll make sure events on both sides consistently mention both.

Reply:
Indeed, we totally missed that point

Reply:
Indeed, we totally missed that point